plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/PreparedQuery.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
typo in commit message for `Separate query rendering from statement creation`.\
s/not/now.